spec/notifiers/publishing_api_notifier_spec.rb

```diff
@@ -9,7 +9,7 @@
       expect(Services.publishing_api).to receive(:put_content).with('69af22e0-da49-4810-9ee4-22b4666ac627', test: :payload)
       expect(Services.publishing_api).to receive(:publish).with('69af22e0-da49-4810-9ee4-22b4666ac627', 'minor')
 
-      PublishingApiNotifier.publish(presenter)
+      PublishingApiNotifier.new.publish(presenter)
```

Comment:
That makes the spec test the `publish` instance method instead of the `self.publish` class method, which seems more relevant? The class method instantiates the presenters and calls the instance method. The latter is the one that's making the calls to the Publishing API so seems more important to test / more likely to break IMO.